test.sh

```diff
@@ -19,8 +19,6 @@ if [[ ${CI} != "true" || (${NODE_ENV} = "production" && ${TRAVIS_PULL_REQUEST} !
   echo "Running local server..."
   exec 3< <(npm run $server)
 
-  sed '/webpack: Compiled successfully.$/q' <&3 ; cat <&3 &
```

Comment:
We need to bring this back but with condition to be fired only for "dev" env
When you run `test.sh` script locally now it tends not to hang tests because it doesn't wait for server to start

Reply:
Makes sense, could you raise a PR?